build/dependencies.yaml

```diff
@@ -75,6 +75,8 @@ dependencies:
       match: quay.io/coreos/etcd
     - path: test/e2e/framework/nodes_util.go
       match: const etcdImage
+    - path: test/utils/image/manifest.go
+      match: configs\[Etcd\] = Config{list\.GcEtcdRegistry, "etcd", "\d+\.\d+.\d+(-(alpha|beta|rc).\d+)?(-\d+)?"}
```

Comment:
Attempted to make this match the etcd versioning scheme in https://github.com/etcd-io/etcd/tags, I'm not sure if it cares about the "-0" on the container image version but seems like no

Reply:
this version will be in the manifest: https://github.com/kubernetes/kubernetes/blob/0c6f1b4e2ac06da94e904e86ad5dcef81b0f8ed0/cluster/images/etcd/Makefile#L40

Reply:
so revision is needed, it is coming from k8s, not from etcd

Reply:
at least in the file I updated for e2e

Reply:
FWIW this tool is just going to check for a substring match with the `version` field after identifying the line that matches this rule. so it's also OK to write a less complex match regex.